docs/ref/contrib/admin/index.txt

```diff
@@ -1296,6 +1296,41 @@ subclass::
     a full count on the table which can be expensive if the table contains a
     large number of rows.
 
+.. attribute:: ModelAdmin.sortable_by
+
+    .. versionadded:: 2.1
+
+    By default, the change list page allows users to interactively sort it by
+    all model fields (and callables that have the ``admin_order_field`` property
+    set) specified by :attr:`list_display`. However, you might want to disable
+    interactive sorting for any/all of them.
+
+    With ``sortable_by`` you can specify a subset of these items. Set
+    ``sortable_by`` to a collection of these same items and they will be
+    available to sort by.
+
+    You can disable customization of such ordering by specifying an empty
+    collection.
+
+    If you need to specify this list dynamically you can implement a
+    :meth:`~ModelAdmin.get_sortable_by` method instead.
+
+    Example::
+
+        class Person(models.Model):
+            first_name = models.CharField(max_length=50)
+            last_name = models.CharField(max_length=50)
+            age = models.PositiveIntegerField()
+            ssn = models.CharField('SSN', max_length=11)
+
+            def full_name(self):
+                  return self.first_name + ' ' + self.last_name
```

Comment:
Please fix indentation (remove two extra spaces), i.e.:
```
                return self.first_name + ' ' + self.last_name
```
instead of
```
                  return self.first_name + ' ' + self.last_name
```